packages/contract/lib/utils/index.js

```diff
@@ -21,23 +21,26 @@ const Utils = {
     return web3Utils.isBN(val) || web3Utils.isBigNumber(val);
   },
 
-  is_tx_params(val) {
+  isTxParams(val) {
     if (!Utils.is_object(val)) return false;
     if (Utils.is_big_number(val)) return false;
 
-    const allowed_fields = {
-      from: true,
-      to: true,
-      gas: true,
-      gasPrice: true,
-      value: true,
-      data: true,
-      nonce: true,
-      privateFor: true
-    };
-
-    for (let field_name of Object.keys(val)) {
-      if (allowed_fields[field_name]) return true;
+    const allowedFields = new Set([
+      "from",
+      "to",
+      "gas",
+      "gasPrice",
+      "value",
+      "data",
+      "nonce",
+      "privateFor",
+      "overwrite"
+    ]);
+
+    for (let fieldName of Object.keys(val)) {
```

Comment:
This `for` loop could be done more succinctly with `Object.keys(val).some(...)`.

Reply:
Yeah that sounds like a good edit!